website_megamenu/static/src/js/content.js

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Odoo
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
This is LGPL. The same in all files.

Also, please, in each file add a little comment in the top pointing to the upstream v13 file where code was imported from. This way, we know better how to compare it with upstream.

Thanks!